bundles/pom.xml

```diff
@@ -484,12 +480,10 @@
     <module>org.openhab.voice.watsonstt</module>
     <module>org.openhab.voice.whisperstt</module>
   </modules>
-
```

Comment:
these are unexpected. Only the change on line 145 is expected.

Reply:
Hm, don't know, but I can see them also in mine fork and in openhab/openhab-addons

Reply:
I do know and they are unexpected :-)
Please revert all changes and only change the line 145. You can copy paste the contents of:
https://raw.githubusercontent.com/openhab/openhab-addons/refs/heads/main/bundles/pom.xml

Reply:
20241014
Done

Reply:
I think this was fxied before, but somehow the unexpected changes came back.